MOM6/patches/MOM_coupler_types.F90.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/MOM6/src/framework/MOM_coupler_types.F90 b/MOM6/src/framework/MOM_coupler_types.F90.new
+index f87b409..124d786 100644
+--- a/MOM6/src/framework/MOM_coupler_types.F90
++++ b/MOM6/src/framework/MOM_coupler_types.F90.new
+@@ -8,7 +8,10 @@ use MOM_couplertype_infra, only : CT_set_diags, CT_send_data, CT_write_chksums,
+ use MOM_couplertype_infra, only : CT_copy_data, CT_increment_data, CT_rescale_data
+ use MOM_couplertype_infra, only : CT_set_data, CT_extract_data, CT_redistribute_data
+ use MOM_couplertype_infra, only : coupler_1d_bc_type, coupler_2d_bc_type, coupler_3d_bc_type
+-use MOM_couplertype_infra, only : ind_flux, ind_alpha, ind_csurf
++use MOM_couplertype_infra, only : ind_flux, ind_deltap, ind_kw, ind_flux0
```

Comment:
Do I understand correctly that the only reason to have these use statements here is to make them available to other modules that use this one?

If so, I would think it's a better practice to have the use statements where they are actually needed. But I assume you are just following what seems to be common practice in MOM?

Reply:
Yeah, this is a layer to avoid directly using routines directly from FMS. It enables FMS1 and FMS2 to be used, despite their different APIs.